CONTRIBUTING.md

```diff
@@ -186,10 +186,6 @@ during development.
   to work with an AWS account that you have access to.  You can use the same user
   that you use for CircleCi.
 
-_Note_: All integration tests are set up to run in `eu-west-*` region.  If you prefer
-to run in a different region you must update the region in each test before running it
-
-
```

Comment:
Considering no other reference in the code to `eu-west` I can only assume that this text above is no longer true.

Reply:
We are amendable to removing this however we cannot remove it based on your assumption.   Would you be able to prove your assumption?

Reply:
Ah I know what happened, the code overwrites some of this stuff when you run it, and that's why grep didn't find it? Anyway, the `eu-west` bits really are still there so I'll revert this.